python/src/opendp/context.py

```diff
@@ -709,24 +709,30 @@ def fix(self, d_in: float, d_out: float, output_measure: Optional[Measure] = Non
         # The delta parameter should be fixed in _cast_measure, and if not, then the search will be impossible here anyways.
         if output_measure == fixed_smoothed_max_divergence():
             def predicate(param):
-                meas = _cast_measure(self.partial(param), output_measure, d_out)
+                meas = _cast_measure(self(param), output_measure, d_out)
                 return meas.map(d_in)[0] <= d_out[0] # type: ignore[index] 
         else:
             def predicate(param):
-                meas = _cast_measure(self.partial(param), output_measure, d_out)
+                meas = _cast_measure(self(param), output_measure, d_out)
                 return meas.check(d_in, d_out)
         
         param = binary_search(predicate, T=T)
         chain = self.partial(param)
         chain.param = param
         return chain
 
-    def __rshift__(self, other: Union[Transformation, Measurement]):
+    def __rshift__(self, other: Union[Transformation, Measurement, PartialConstructor]):
         # partials may be chained with other transformations or measurements to form a new partial
-        # TODO: Can we exercise this?
-        if isinstance(other, (Transformation, Measurement)): # pragma: no cover
-            return PartialChain(lambda x: self.partial(x) >> other)
+        if isinstance(other, (Transformation, Measurement, PartialConstructor)):
+            return PartialChain(lambda x: self(x) >> other)
 
+        print(self, other)
```

Comment:
Correction: all `raise` lines are ignored during the coverage check.
```
exclude_also =
  # TODO: Improve coverage and shorten this list!
  ^\s*raise\b
  os\.environ
  ^\s*\.\.\.\s*$
```
so, no `pragma` is needed here... although it obviously would be good to expand coverage and make sure we reach this when appropriate.